cliquet/cache/postgresql/__init__.py

```diff
@@ -40,6 +40,9 @@ class PostgreSQL(PostgreSQLClient, CacheBase):
         boost performances and bound memory usage (*work_mem per connection*).
 
     """
+
+    __metaclass__ = statsd.CacheTimer
```

Comment:
Instead of requiring every cache/storage backend to have this metaclass, couldn't we do something directly in ``statsd.setup()`` ?

Like : 

```
def decorate_methods(backend, prefix=''):
      members = dir(backend)
      for key, value in members.items():
          ...
          setattr(backend, key, decorated)

statds.decorate_methods(config.registry.storage, prefix='storage')
statds.decorate_methods(config.registry.cache, prefix='cache')
```

This way we set it up only if activated, and we don't have to put on every backend class.

What do you think ?